### ISSUE TITLE
Add categories to the requirements specification

### ISSUE THREAD

**Issue by burnhamd** [innolitics/rdm#46]
It would be helpful to have categories in the requirements specification.  The the requirements could be grouped by category with a header as the category name.  For it is helpful to have a category for each software module and then have requirements for that module.  Or to separate function and performance.

**Comment by johndgiese** [comment]
@burnhamd I think this is a good idea---we use requirements categories on several of our client projects. RDM's templating system is pretty generic, and can provide requirements categories. I attached a git commit diff that shows how you would modify the RDM v0.8 (just released) init_files so as to accomplish this. I also included a zip archive that contains git repository that showing off as much.

Let me know if you have any other questions or comments about this issue! If I don't hear from you in a few days, I will go ahead and close this.

[add_requirement_categories_diff.txt](https://github.com/innolitics/rdm/files/4565914/add_requirement_categories.txt)
[rdm-example.zip](https://github.com/innolitics/rdm/files/4565915/rdm-example.zip)
